Src/main.c

```diff
@@ -613,99 +626,125 @@ preprocess(char *a, char *b, int a_tmp)
-	printf("\t      options following a -search are used to compile and run the verifier pan\n");
-	printf("\t	    valid options that can follow a -search argument include:\n");
-	printf("\t	    -bfs	perform a breadth-first search\n");
-	printf("\t	    -bfspar	perform a parallel breadth-first search\n");
-	printf("\t	    -dfspar	perform a parallel depth-first search, same as -DNCORE=4\n");
-	printf("\t	    -bcs	use the bounded-context-switching algorithm\n");
-	printf("\t	    -bitstate	or -bit, use bitstate storage\n");
-	printf("\t	    -biterateN,M use bitstate with iterative search refinement (-w18..-w35)\n");
-	printf("\t			perform N randomized runs and increment -w every M runs\n");
-	printf("\t			default value for N is 10, default for M is 1\n");
-	printf("\t			(use N,N to keep -w fixed for all runs)\n");
-	printf("\t			(add -w to see which commands will be executed)\n");
-	printf("\t			(add -W if ./pan exists and need not be recompiled)\n");
-	printf("\t	    -swarmN,M like -biterate, but running all iterations in parallel\n");
-	printf("\t	    -link file.c  link executable pan to file.c\n");
-	printf("\t	    -collapse	use collapse state compression\n");
-	printf("\t	    -noreduce	do not use partial order reduction\n");
-	printf("\t	    -hc  	use hash-compact storage\n");
-	printf("\t	    -noclaim	ignore all ltl and never claims\n");
-	printf("\t	    -p_permute	use process scheduling order random permutation\n");
-	printf("\t	    -p_rotateN	use process scheduling order rotation by N\n");
-	printf("\t	    -p_reverse	use process scheduling order reversal\n");
-	printf("\t	    -rhash      randomly pick one of the -p_... options\n");
-	printf("\t	    -ltl p	verify the ltl property named p\n");
-	printf("\t	    -safety	compile for safety properties only\n");
-	printf("\t	    -i	    	use the dfs iterative shortening algorithm\n");
-	printf("\t	    -a	    	search for acceptance cycles\n");
-	printf("\t	    -l	    	search for non-progress cycles\n");
-	printf("\t	similarly, a -D... parameter can be specified to modify the compilation\n");
-	printf("\t	and any valid runtime pan argument can be specified for the verification\n");
-	printf("\t-S1 and -S2 separate pan source for claim and model\n");
-	printf("\t-s print send events\n");
-	printf("\t-T do not indent printf output\n");
-	printf("\t-t[N] follow [Nth] simulation trail, see also -k\n");
-	printf("\t-Uyyy pass -Uyyy to the preprocessor\n");
-	printf("\t-uN stop a simulation run after N steps\n");
-	printf("\t-v verbose, more warnings\n");
-	printf("\t-w very verbose (when combined with -l or -g)\n");
-	printf("\t-[XYZ] reserved for use by xspin interface\n");
-	printf("\t-V print version number and exit\n");
-	alldone(1);
-}
+#define OPT_KEY_RUN 1
+#define OPT_KEY_PRETTY_PRINT 2
+#define OPT_KEY_LTL 3
+#define OPT_KEY_LINK 4
+#define OPT_KEY_REPLAY 5
+#define OPT_KEY_SIMULATE 6
+#define OPT_KEY_SEARCH 7
+#define OPT_KEY_S1 8
+#define OPT_KEY_S2 9
```

Comment:
These defines are only used as unique values in parsing the long command-line arguments, because argp needs a unique `char` value to separate everything. These don't need to be printable ASCII, and the short options use their single letters already